lib/nexmo_developer/app/services/translator/file_translator.rb

```diff
@@ -0,0 +1,48 @@
+module Translator
+  class FileTranslator
+    attr_reader :doc_path
+
+    def initialize(doc_path)
+      @doc_path = doc_path
+    end
+
+    def translation_requests
+      doc = load_doc
+      requests = []
+
+      I18n.available_locales.each do |locale|
+        requests << Translator::TranslationRequest.new(locale: locale, frequency: frequency_calculator(doc), path: doc_path)
+      end
+
+      requests
+    end
+
+    def load_doc
+      YAML.safe_load(File.read("#{Rails.configuration.docs_base_path}/_documentation/en/#{doc_path}"))
```

Comment:
can you memoize this? and maybe rename it to `#frontmatter`?

Reply:
👍